reviewing/plugin-commands-licenses/src/outputRenderer.ts

```diff
@@ -106,10 +106,21 @@ function renderLicensesTable (
   return table(
     [
       columnNames,
-      ...sortLicensesPackages(licensePackages).map((licensePkg) => {
-        return columnFns.map((fn) => fn(licensePkg))
-      }),
+      ...deduplicateTable(
+        sortLicensesPackages(licensePackages)
+          .map((licensePkg) => columnFns.map((fn) => fn(licensePkg)))
+      ),
     ],
     TABLE_OPTIONS
   )
 }
+
+function deduplicateTable (table: string[][]): string[][] {
+  const result: string[][] = []
+  const rowEqual = (a: string[], b: string[]) => a[0] === b[0] && a[1] === b[1]
```

Comment:
0 and 1 are the columns, right? Maybe use some constants to name the columns.

Reply:
I have replaced index numbers with field names of `LicensePackage`: https://github.com/pnpm/pnpm/pull/7528/commits/f5ed0e0e80bf7e6b462169c71d602cee59d81a8a